### PR TITLE
Upgrade actions/stale version from v2.0.0 to v9

### PR DESCRIPTION
**Pull request by hawl1**
No one wants their Github Actions to not work, especially if its in a big project.

### CODE REVIEW

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ jobs:
         stale-pr-label: 'status: stale'
         days-before-stale: 90
         days-before-close: 30
-    - uses: actions/stale@v2.0.0
+    - uses: actions/stale@v9.0.0
```
Comment (tych0) [inline .github/workflows/stale.yml:20]: Can we use v9 here to get any point releases? (i.e. without the .0.0?)

Comment (hawl1) [reply .github/workflows/stale.yml:20]: yes, we can. let me change it real quick...

Comment (hawl1) [reply .github/workflows/stale.yml]: Alright, done.

**Comment by tych0** [comment]
Hmm. I tried to squash this and it closed the PR?

**Comment by tych0** [comment]
Ok, I dunno what happened there, but I manually pushed it to master: 3e7284b0514e47eb700900fcc037812b86b8481e

Thanks!

**Comment by hawl1** [comment]
github being github lmao

**Pull request closed without merging**